ansible/roles/owncloud/tasks/ldap.yml

```diff
@@ -14,6 +14,9 @@
   register: owncloud__register_ldap_default_config_exits
   become: True
   become_user: '{{ owncloud__app_user }}'
+  #ignore_errors: True
```

Comment:
What is the purpose of this commented out line?

Reply:
just left over. fixed it.